pkg/apis/kubermatic/v1/configuration.go

```diff
@@ -61,7 +61,16 @@ type KubermaticConfiguration struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec KubermaticConfigurationSpec `json:"spec,omitempty"`
+	Spec   KubermaticConfigurationSpec   `json:"spec,omitempty"`
+	Status KubermaticConfigurationStatus `json:"status,omitempty"`
+}
+
+// KubermaticConfigurationStatus stores status information about a KubermaticConfiguration.
+type KubermaticConfigurationStatus struct {
+	// KubermaticVersion current kubermatic version.
+	KubermaticVersion string `json:"kubermaticVersion"`
+	// KubermaticEdition current kubermatic edition , i.e. Community Edition or Enterprise Edition.
+	KubermaticEdition string `json:"kubermaticEdition"`
```

Comment:
Please mark those fields as omitempty, as mandatory fields in a status will not work easily.